server/@types/bapv.d.ts

```diff
@@ -1,5 +1,13 @@
 import { InmateDetail, VisitBalances } from '../data/prisonApiTypes'
 
+declare global {
```

Comment:
Needed to allow adding `id` to the request object.

Reply:
would it be better to add the type override [here](https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/blob/9b21c7f7076e16184404856b11404881af2cc90a/server/%40types/express/index.d.ts#L19)

Reply:
Done 👍